superset/assets/javascripts/explore/components/EmbedCodeButton.jsx

```diff
@@ -29,7 +30,7 @@ export default class EmbedCodeButton extends React.Component {
   generateEmbedHTML() {
     const srcLink = (
       window.location.origin +
```

Comment:
it's strange to use string templates and concatenation. can we just use the template?

Reply:
if all use template, it will break lint rule. so i have to make it multiple lines.